### PR TITLE
Created controller for start items

### PR DESCRIPTION
**Pull request by VladyslavAnokhin**
## Changes

Created controller which starts new items from current processed group.

[JIRA Ticket](https://jira.ouroath.com/browse/OMSDK-2014)

@OathAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: sources/advertisements/VRM New Core/Controllers/StartVRMGroupProcessingController.swift [hunk 1]
```diff
@@ -23,6 +23,5 @@ final class StartVRMGroupProcessingController {
         let nextGroup = groupsQueue.first else { return }
         
         dispatch(VRMCore.startGroupProcessing(group: nextGroup))
-
```
Comment (AndriiMoskvin) [inline sources/advertisements/VRM New Core/Controllers/StartVRMGroupProcessingController.swift:26]: Important change!

**Review by AndriiMoskvin — DISMISSED** [review]

File: sources/advertisements/VRM New Core/Controllers/VRMItemControllerTest.swift [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        urlItem = VRMCore.Item.url(url, metaInfo)
+    }
+    
+    override func tearDown() {
+        recorder.verify {}
+        recorder.clean()
+        super.tearDown()
+    }
+    
+    func testStartItemsFetcing() {
```
Comment (AndriiMoskvin) [inline sources/advertisements/VRM New Core/Controllers/VRMItemControllerTest.swift]: typo here

File: sources/advertisements/VRM New Core/Controllers/VRMItemControllerTest.swift [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    
+    func testDoubleDispatch() {
+        let dispatch = recorder.hook("testDoubleDispatch") { targetAction, recordeAction -> Bool in
+            switch(targetAction, recordeAction) {
+            case let ( VRMCore.StartItem.parsing(targetItem, targetVast, _),
+                       VRMCore.StartItem.parsing(recordedItem, recordedVast, _) ):
+                return targetVast == recordedVast && recordedItem == targetItem
+            default: return false
+            }
+            } as (PlayerCore.Action) -> ()
```
Comment (AndriiMoskvin) [inline sources/advertisements/VRM New Core/Controllers/VRMItemControllerTest.swift]: `let dispatch: (PlayerCore.Action) -> () = ...` is more readable for me

**Comment by VladyslavAnokhin** [comment]
@AndriiMoskvin Ready for review

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**